client/main.js

```diff
@@ -34,10 +35,10 @@ window.addEventListener( 'beforeunload', ( event ) => {
 const rootEl = document.getElementById( 'wc-connect-admin-container' );
 
 let render = () => {
-	const WCCShippingSettings = require( 'views/shipping' );
+	const RootView = require( 'views/root/' + rootView );
```

Comment:
WebPack loads in the bundle all the modules that match the pattern `views/root/*` , so it's basically as expected.